kolibri/core/auth/serializers.py

```diff
@@ -63,6 +66,26 @@ def save(self, **kwargs):
 
     def validate(self, attrs):
         username = attrs.get("username")
+        username_validator = RegexValidator(
```

Comment:
This could probably be defined in module scope? Could even be defined this in models.py and used there and here.

Reply:
Done